software/firmware/europi.py

```diff
@@ -622,6 +622,64 @@ def value(self, value):
             self.off()
 
 
+class Thermometer:
+    """
+    Wrapper for the temperature sensor connected to Pin 4
+    Reports the module's current temperature in Celsius.
+    If the module's temperature sensor is not working correctly, the temperature will always be reported as None
+    """
+
+    # Conversion factor for converting from the raw ADC reading to sensible units
+    # See Raspberry Pi Pico datasheet for details
+    TEMP_CONV_FACTOR = 3.3 / 65535
+
+    def __init__(self):
+        # The Raspberry Pi Pico 2's temperature sensor doesn't with some older

```

Comment:
[Spelling]
Assume this should read "sensor doesn't *work*  with some older.."

Reply:
Correct. Fixed.